pkgs/tools/networking/opensnitch/ui.nix

```diff
@@ -48,6 +50,8 @@ python3Packages.buildPythonApplication rec {
   dontWrapQtApps = true;
   makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
 
+  doCheck = false; # checks fail for now
```

Comment:
it would be nice for sandbox incompatible tests to be disabled. But turning off all tests isn't great.

Reply:
I think _all_ the tests for opensnitch-ui are currently sandbox-incompatible.  At least; that was the impression I got from their failures when I ran them outside an X session by hand.